tests/components/p1_monitor/test_sensor.py

```diff
@@ -125,9 +121,9 @@ async def test_phases(
     assert entry.unique_id == f"{entry_id}_phases_power_consumed_phase_l1"
     assert state.state == "315"
     assert state.attributes.get(ATTR_FRIENDLY_NAME) == "Power Consumed Phase L1"
-    assert state.attributes.get(ATTR_STATE_CLASS) == STATE_CLASS_MEASUREMENT
+    assert state.attributes.get(ATTR_STATE_CLASS) is SensorStateClass.MEASUREMENT
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == POWER_WATT
-    assert state.attributes.get(ATTR_DEVICE_CLASS) == DEVICE_CLASS_POWER
+    assert state.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.POWER
```

Comment:
```suggestion
    assert state.attributes.get(ATTR_DEVICE_CLASS) is SensorDeviceClass.POWER
```
Why not **is** also for the `SensorDeviceClass`?

Reply:
I have previously tried this, but several state attributes are strings including this one.

Reply:
See [here](https://github.com/home-assistant/core/pull/61989#discussion_r770447331) for details on why this currently needs the string comparison.